e2e_testing/torchscript/basic.py

```diff
@@ -666,6 +666,26 @@ def _LogSoftmaxModule_basic(module, tu: TestUtils):
     module.forward(torch.randn(3, 2, 4))
 
 # ==============================================================================
+
+class HardsigmoidModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.hardsigmoid(x)
+
+
+@register_test_case(module_factory=lambda: HardsigmoidModule())
+def HardsigmoidModule_basic(module, tu: TestUtils):
+    module.forward(torch.tensor([[4.0, -5.0, 3.0], [2.9, -1.5, -3.0]]))
```

Comment:
is there a reason the input tensor is not randomly generated?

Reply:
Since this is a piece-wise function, I wanted the inputs to be at all parts of the function. But I can add one more test case having random values for better coverage. Thanks :)